bbq/compiler/compiler.go

```diff
@@ -229,6 +229,16 @@ func (c *Compiler[_]) addStringConst(str string) *constant {
 	return c.addConstant(constantkind.String, []byte(str))
 }
 
+func (c *Compiler[_]) intConstLoad(i int64) {
+	constant := c.addIntConst(i)
+	c.codeGen.Emit(opcode.InstructionGetConstant{ConstantIndex: constant.index})
+}
+
+func (c *Compiler[_]) addIntConst(i int64) *constant {
+	data := leb128.AppendInt64(nil, i)
+	return c.addConstant(constantkind.Int, data)
+}
```

Comment:
Nice! Could we maybe use this in `VisitIntegerExpression` while at it?

Reply:
Good idea! Added in 3121eda24c101d3de9be148ec32184a28a4b451f